test/Microsoft.AspNetCore.Razor.Tools.Test/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "testRunner": "xunit",
+  "dependencies": {
+    "dotnet-test-xunit": "2.2.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.1.0-*",
```

Comment:
Talked offline and going to target the RTM version of Mvc.TagHelpers here (in addition to modifying universe build) to prevent cyclical builds.